packages/blocks/src/lib/navigation-map/components/robot-marker.svelte

```diff
@@ -1,14 +1,23 @@
 <script lang="ts">
-import { theme } from '@viamrobotics/prime-core/theme';
 import { MapLibreMarker, type LngLat } from '$lib';
 
+export let robotIcon = document.createElement('div');
+robotIcon.className = 'robot-marker';
+robotIcon.style.backgroundImage = `url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24' fill='green'%3E%3Cpath d='M12,2L4.5,20.29L5.21,21L12,18L18.79,21L19.5,20.29L12,2Z' /%3E%3C/svg%3E")`;
```

Comment:
I'm a little worried about the lack of contrast of this icon on green backgrounds. Could we get design to produce us one that would work a bit better? Maybe something as simple as green but with a white outline would be better